gio/src/auto/initable.rs

```diff
@@ -0,0 +1,67 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::Cancellable;
+use glib::object::IsA;
+use glib::translate::*;
+use std::fmt;
+use std::ptr;
+
+glib::wrapper! {
+    #[doc(alias = "GInitable")]
+    pub struct Initable(Interface<ffi::GInitable, ffi::GInitableIface>);
+
+    match fn {
+        type_ => || ffi::g_initable_get_type(),
+    }
+}
+
+impl Initable {
+    //#[doc(alias = "g_initable_new")]
+    //pub fn new<P: IsA<Cancellable>>(object_type: glib::types::Type, cancellable: Option<&P>, error: &mut glib::Error, first_property_name: Option<&str>, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> glib::Object {
```

Comment:
You could implement this manually around `glib::Object::new()` and `Initable::init()` if you want. That should be like 5 lines of code :)